src/appshell/view/notationstatusbarmodel.cpp

```diff
@@ -314,9 +325,9 @@ void NotationStatusBarModel::zoomOut()
     dispatch("zoomout");
 }
 
-void NotationStatusBarModel::handleAction(const QString& actionCode)
+void NotationStatusBarModel::handleMenuItem(const QString& itemId)
 {
-    dispatch(codeFromQString(actionCode));
+    dispatch(codeFromQString(itemId));
```

Comment:
I believe you need to explicitly find the item by itemId and get actionCode from it